jsons/Policies.json

```diff
@@ -427,6 +428,8 @@
                 "uniques": [
                     "[+100]% Production when constructing [Archaeologist] units [in all cities]",
                     "[+1 Culture] from every [Great Work]",
+                    "[+2 Culture, +2 Science] from every [Archaeological Dig]",
+                    //can't add "Shows Hidden Antiquity Sites", buff then
```

Comment:
Would prefer to not give an extra buff. Perhaps this should be a suggestion to Unciv itself to allow for this

Reply:
removed